### PR TITLE
[release] 2.53.0 Website Updates

### PR DESCRIPTION
**Pull request by jrmccluskey**
Adds 2.53.0 release information to the beam website. DO NOT MERGE until release is approved 

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by jrmccluskey** [comment]
R: @damccorm

**Review by svetakvsundhar — APPROVED** [review]

File: website/www/site/content/en/blog/beam-2.53.0.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+See the [download page](/get-started/downloads/) for this release.
+
+<!--more-->
+
+For more information on changes in 2.53.0, check out the [detailed release notes](https://github.com/apache/beam/milestone/17).
+
+## Highlights
+
+* Python streaming users that use 2.47.0 and newer versions of Beam should update to version 2.53.0, which fixes a known issue: ([#27330](https://github.com/apache/beam/issues/27330)).
+* New highly anticipated feature Y added to Java SDK ([#Y](https://github.com/apache/beam/issues/Y)).
```
Comment (damccorm) [inline website/www/site/content/en/blog/beam-2.53.0.md:33]: We should remove this

Comment (damccorm) [reply website/www/site/content/en/blog/beam-2.53.0.md:33]: Also probably needs removed from CHANGES.md

**Review by damccorm — APPROVED** [review]
LGTM once pending comments are addressed

File: website/www/site/content/en/blog/beam-2.53.0.md (hunk 1 above)
Comment (jrmccluskey) [reply website/www/site/content/en/blog/beam-2.53.0.md]: Done

**Review by damccorm — APPROVED** [review]

**Pull request merged**